flags.go

```diff
@@ -225,8 +224,8 @@ func newApp() (app *cli.App) {
 
 			cli.DurationFlag{
 				Name:  "max-retry-duration",
-				Value: 30 * time.Second,
-				Usage: "The operation will be retried till the value of max-retry-duration.",
+				Value: -1 * time.Second,
+				Usage: "The operation will be retried till the value of max-retry-duration. This flag is currently unused.",
```

Comment:
Should we also log a warning when this flag is set? (now that the default is changed, we can identify when the value is different than -1 and log a warning)

Reply:
Yes, I'll raise separate PR for that for info logs

Reply:
Yeah, I was thinking the same, let's do the same as we did for that debug flag.
`And if we are not warning then we can keep 30s as default too.` --> -1 will make sense I guess as we are not using this flag anywhere.

Reply:
okay, let's just use "This flag is currently unused." as description